cluster-autoscaler/dynamic/config.go

```diff
@@ -0,0 +1,86 @@
+package dynamic
```

Comment:
there's already a `cluster-autoscaler/config` package, wondering if this should all go in there, or if `dynamic` should be a subpackage of config.